### ISSUE TITLE
compile error: unknown variant `metadata`, expected one of `abi`, `evm.methodIdentifiers`, `storageLayout`, `ast`, `irOptimized`, `evm.legacyAssembly`

### ISSUE THREAD

**Issue by itinance** [matter-labs/hardhat-zksync#244]
When adding `hardhat-zksync`-libraries to an existing hardhat-project, I face the following compiler error:

```
Error: Command failed: ~/Library/Caches/hardhat-nodejs/compilers-v2/zksolc/zksolc-v1.3.5 --standard-json    --solc ~/Library/Caches/hardhat-nodejs/compilers-v2/macosx-amd64/solc-macosx-amd64-v0.8.17+commit.8df45f5f
unknown variant `metadata`, expected one of `abi`, `evm.methodIdentifiers`, `storageLayout`, `ast`, `irOptimized`, `evm.legacyAssembly` at line 1 column 218856
```

This happens once 
```
import "@matterlabs/hardhat-zksync-solc";
```

will be added to `hardhat.config.ts`

Has someone already faced this?

**Comment by coffiasd** [comment]
Error: Command failed: C:\Users\75929\AppData\Local\hardhat-nodejs\Cache\compilers-v2\zksolc\zksolc-v1.3.5 --standard-json    --solc C:\Users\75929\AppData\Local\hardhat-nodejs\Cache\compilers-v2\windows-amd64\solc-windows-amd64-v0.8.0+commit.c7dfd78e.exe
unknown variant `metadata`, expected one of `abi`, `evm.methodIdentifiers`, `storageLayout`, `ast`, `irOptimized`, `evm.legacyAssembly` at line 1 column 628

I also encountered the same problem on win

**Comment by edikger** [comment]
use zksolc 1.3.8 version

**Comment by coffiasd** [comment]
I swith zksolc version to v1.2.0 to fix this issue

**Comment by mpopovac-txfusion** [comment]
Fixed with `hardhat-zksync-solc@0.3.16` release.
